presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/JdbcMetadataConfig.java

```diff
@@ -46,4 +49,18 @@ public JdbcMetadataConfig setAllowAggregationPushdown(boolean allowAggregationPu
         this.allowAggregationPushdown = allowAggregationPushdown;
         return this;
     }
+
+    @Min(0)
```

Comment:
1 ? 

0 is kind if unsupported edge case, because `Domain.simplify(0)` does not return `Domain.all`